ui/repl.c

```diff
@@ -186,9 +187,14 @@ void parse_opts(int *argcp, char ***argvp)
             break;
         case 'p': // procs
             errno = 0;
-            jl_options.nprocs = strtol(optarg, &endptr, 10);
-            if (errno != 0 || optarg == endptr || *endptr != 0 || jl_options.nprocs < 1)
-                jl_errorf("julia: -p,--procs=<n> must be an integer >= 1\n");
+            if (!strcmp(optarg,"auto")) {
+                jl_options.nprocs = jl_cpu_cores();
```

Comment:
If this is spinning up N additional workers, shouldn't this be `jl_cpu_cores() - 1`?